diktat-rules/pom.xml

```diff
@@ -67,6 +72,73 @@
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.jetbrains.kotlin</groupId>
+                <artifactId>kotlin-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>build-generated-code</id>
```

Comment:
Is this execution really needed? The generated file is stored under `src/main/kotlin` which is already one of source sets.

Reply:
First we need to compile, so that exec can see WarningsGeneration, otherwise ClassNotFoundException. Then we need to compile WarningNames

Reply:
I see, so `WarningsGeneration` is compiled during the usual `compile` phase, then generated file needs to be compiled before running tests, where it is used.

Reply:
yes